chore(http): Refresh headers docs (d-k) (#36075)
mdn/content: files/en-us/web/http/headers/expires/index.md

{{HTTPSidebar}}

The HTTP **`Expires`** {{Glossary("response header")}} contains the date/time after which the response is considered expired in the context of [HTTP caching](/en-US/docs/Web/HTTP/Caching).

> Invalid expiration dates with value `0` represent a date in the past and mean that the resource is already expired.

Reviewer: Because invalid indicates perhaps incorrect usage, maybe

```suggestion
The value `0` is used to represent a date in the past, indicating the resource has already expired.
```

    </tr>
    <tr>
      <th scope="row">{{Glossary("Forbidden header name")}}</th>
      <td>No</td>
    </tr>
    <tr>
      <th scope="row">
        {{Glossary("CORS-safelisted response header")}}
      </th>
      <td>Yes</td>
    </tr>
  </tbody>
</table>

## Syntax

```http
Expires: <day-name>, <day> <month> <year> <hour>:<minute>:<second> GMT
```

## Directives

- `<day-name>`

>   - : One of "Mon", "Tue", "Wed", "Thu", "Fri", "Sat", or "Sun" (case-sensitive).

Reviewer: As previously, I prefer (generally)
```suggestion
  - : One of `Mon`, `Tue`, `Wed`, `Thu`, `Fri`, `Sat`, or `Sun` (case-sensitive).
```

Author: done in cbbafade262a59b230140907bae499dc3dac0f66

- `<day>`

>   - : 2 digit day number, e.g., "04" or "23".

Reviewer: W.r.t comment above https://github.com/mdn/content/pull/36075/files#r1815794338 note that here I would NOT update the examples to code style. Just the selectable values.

Author: done in cbbafade262a59b230140907bae499dc3dac0f66
